### ISSUE TITLE
[Visual Requirement – BotKit – Docs]: Luminosity Ratio is Less than 4.5:1 for “twilio_sms_bot.js” link present in “Twilio SMS” link.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#10]
**User Experience:**
Low Vision users will get effected to navigate, if Luminosity Ratio is Less than 4.5:1 for “twilio_sms_bot.js” link present in “Twilio SMS” link.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Colour Contrast Analyzer (CCA)

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Open Colour Contrast Analyzer tool.
Step 3: Navigate to “Docs” link and select it.
Step 4: Navigate to “Twilio SMS” link and select it.
Step 5: Navigate to “twilio_sms_bot.js” link and Verify the Luminosity is greater than 4.5:1 or not.

**Actual Result:**
Luminosity Ratio is Less than 4.5:1 for “twilio_sms_bot.js” link present in “Twilio SMS” link.

**Expected Result:**
Luminosity Ratio should be Greater than or Equal to 4.5:1 for “twilio_sms_bot.js” link present in “Twilio SMS” link.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
![MAS1 4 3_BotKit_Docs_Twilo SMS](https://user-images.githubusercontent.com/48272005/54471497-d5196080-47df-11e9-825f-607bbc521912.png)

**Comment by benbrown** [comment]
fixed

**Comment by v-gasikk** [comment]
Issue is Fixed. Hence Closing the bug.
Now Luminosity Ratio is greater than 4.5:1 for “twilio_sms_bot.js” link
